jenkins_github_checkwarn.sh

```diff
@@ -88,7 +88,7 @@ if [ -n "${GERRIT_PROJECT-}" ]; then
   set -u
 fi
 
-if [ -x ./ci/patch_src_in_place ]
+if [ -f ./ci/patch_src_in_place ]
```

Comment:
What is `-x` anyway?

Reply:
File exists and is executable

Reply:
executable.  I **think** the issue is that the file is mode 775 but the owner/group is different, so whilst the file is executable the bash test is failing, and the script isn't being run.

Reply:
I don't think `[ -x ... ]` checks owner/group, FWIW.